changelogs/dev-pnpm

```diff
@@ -0,0 +1,4 @@
+Significance: patch
```

Comment:
Ignore the location of this comment 😅 

Could you link the P2 shortlink in the PR description please? p90Yrv-2OV-p2

Reply:
Sure. 🙂  I totally forgot to add it.